LibertyDSNP/spec
# Feat/convert mdbook/153

Problem
=======
## Acceptance Criteria
- [ ] When I click on menu item without a page link, it should expand the first nested menu
- [ ] I can click on the name or the arrow to expand the nested menu
- [ ] The menu item without a page link looks the same as the other items

Link to GitHub Issue(s): https://github.com/LibertyDSNP/spec/issues/153

Solution
========
Update theme files to set same color scheme for linkless (draft) menu item, also set toggling event listener for draft type div in sidebar menu

Change summary:
---------------
- [ ] Updated theme
- [ ] Updated book.js to add toggle for draft menu items

Steps to Verify:
----------------
1. Run npm run build
1. Browser  will open with mdbook then click "Draft Specification" and "Associated Types" (in Activity Content) they should work same as any other menu toggel/look and feel
